From 90dee7da8ed96ea6e2ade2dd0f085bfb575449b3 Mon Sep 17 00:00:00 2001
From: Alaric Senat <dev.asenat@posteo.net>
Date: Tue, 13 Sep 2022 19:48:09 +0200
Subject: [PATCH] win32: remove wrong safe wrappers

Theses wrapper were too naively written and are causing a lot of
regressions on windows.

This has been addressed upstream [1]. And is simply reverted for now,
waiting for a new release.

[1] https://github.com/pupnp/pupnp/issues/411
---
 ixml/inc/posix_overwrites.h | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/ixml/inc/posix_overwrites.h b/ixml/inc/posix_overwrites.h
index a9c49e3f..7e37a355 100644
--- a/ixml/inc/posix_overwrites.h
+++ b/ixml/inc/posix_overwrites.h
@@ -11,11 +11,5 @@
 	#define strncasecmp strnicmp
 	#define strnicmp _strnicmp
 
-	/* Secure versions of functions */
-	#define strcat(arg1, arg2) strcat_s(arg1, sizeof(arg1), arg2)
-	#define strcpy(arg1, arg2) strcpy_s(arg1, _countof(arg1), arg2)
-	#define strncpy(arg1, arg2, arg3) strncpy_s(arg1, arg3, arg2, arg3)
-	#define sprintf(arg1, ...) sprintf_s(arg1, sizeof(arg1), __VA_ARGS__)
-
 #endif /* _WIN32 */
 #endif /* POSIX_OVERWRTIES_H */
-- 
2.35.1

